lib/virt_autotest/utils.pm

```diff
@@ -203,6 +203,12 @@ sub check_libvirtd {
     record_info("Libvirtd Daemon has been checked!");
 }
 
+# libvirt daemon for yast virtualization
+sub manage_libvirt_yast_virtualization {
```

Comment:
It seems to me that there is no need to make a library function for the two lines. Better to just put the lines directly at the calling place. Besides what's the purpose of `script_run('rpm -q libvirt-libs')`? It won't fail/stop test if libvirt-libs is not installed. Just for debug info?
At last would you please confirm if the host that is installed in the job would always have the required virtualization packages/patterns installed? I remember in earlier comments, you met problems of no available virt packages?

Reply:
Yes, you are correct. I will change that, initially, I thought `creating function > direct placement` but for creating a lib function we can avoid as you said two lines only and I can achieve that as Wayne said.

The use of `script_run('rpm -q libvirt-libs')` in this context is primarily for informational purposes, and you're correct that it won't fail the test. It's useful to know whether the required package is installed or not, especially for debugging and troubleshooting for future and this will be addressed in Wayne's suggestion.